### ISSUE TITLE
How to enable prometheus metrics with the helm chart?

### ISSUE THREAD

**Issue by katepangLiu** [traefik/traefik-helm-chart#422]
With stable/traefik ,  we can `--set metrics.prometheus.enabled=true` to enable prometheus metrics, but here I can't find the relevant option.

**Comment by jakubhajek** [comment]
hello @katepangLiu 

You can enable Prometheus metrics by passing the following parameter to Helm:

`helm upgrade  --set="additionalArguments={--metrics.prometheus=true}" traefik traefik/traefik`

You can also create a values.yaml file and add all argumentes e.g. 

```yaml
additionalArguments:
  - "--metrics.prometheus=true"
```
then just run:

`helm upgrade traefik traefik/traefik -f values.yaml`

**Comment by StianOvrevage** [comment]
I found this confusing as well.

Not finding anything about metrics in the Helm chart I looked up the environment variables as that seemed like the next level deeper for configuration ( https://doc.traefik.io/traefik/reference/static-configuration/env/ ) and set:

```
env:
 - name: TRAEFIK_METRICS_PROMETHEUS
   value: "true"
```

That had no effect.

Then looking at https://doc.traefik.io/traefik/observability/metrics/overview/ it seems I have to enable metrics itself with `--metrics=true` and then `--metrics.prometheus=true`, so I added

```
additionalArguments:
  - "--metrics=true"
```

That caused the container to crash loop since only enabling metrics without a defined exporter is apparently invalid. And the environment variable does nothing.

I then removed the env var and went for arguments instead:

```
additionalArguments:
  - "--metrics=true"
  - "--metrics.prometheus=true"
```

And it's working.

However after reading this thread I thought about trying with just

```
 additionalArguments:
  - "--metrics.prometheus=true"
```

And that also works!

I'm left with a feeling of confusion. Where and how is the correct way to configure Traefik deployed via Helm to K8s? How are arguments either depending, excluding or implying others? And what about the env vars?
